Submitty/Submitty
# Course materials date picker timezone handling

Closes #4065

Address issues outlined in #4065.

**To Test**
- Set your PC timezone to the same time zone as the server. *This is default behavior for vagrant users in New York*
- Navigate to ```Course Materials```
- Upload some files and observe the behavior of the date time picker when using it
- Change your PC timezone to some timezone outside of server timezone
- Refresh the page
- Observe the behavior of the date time picker
